Live/Classes/MainScene.h

```diff
@@ -23,6 +26,7 @@ class MainScene : public cocos2d::Scene {
         keys;
     Player* m_player;
     HUD* m_hud;
+    std::vector<Item*> m_items;
```

Comment:
specify that these are the items spread around the world since it isn't obvious

Reply:
Done